bin/install.sh

```diff
@@ -1,13 +1,28 @@
 #!/bin/bash
+set -e
+
 export BUILD_HARNESS_ORG=${1:-mintel}
 export BUILD_HARNESS_PROJECT=${2:-build-harness}
 export BUILD_HARNESS_BRANCH=${3:-master}
+export BUILD_HARNESS_TAG=${4:-latest}
```

Comment:
We could track the latest version tag instead of "latest". Also it would be nice for performance to not pull the entire build-harness history. Something like this:
```sh
# Get latest version tag.
export BUILD_HARNESS_BRANCH="$4"
if [ -z "$BUILD_HARNESS_BRANCH" ]; then
    BUILD_HARNESS_BRANCH="$(git ls-remote --tags "$GITHUB_REPO" 2>/dev/null | cut -d/ -f3 | sort -rV | head -1)"
fi

# Initial clone, only pulling one commit.
git clone -b "${BUILD_HARNESS_BRANCH}" --depth=1 --no-tags "${GITHUB_REPO}" 2>/dev/null

# Update to latest version tag, again only pulling one commit.
git fetch --depth=1 --no-tags origin "${BUILD_HARNESS_BRANCH}" 2>/dev/null
git checkout FETCH_HEAD 2>/dev/null
```

Reply:
#37